examples/shaders/resources/shaders/glsl120/raymarching.fs

```diff
@@ -0,0 +1,431 @@
+//Ported by CDM to work on older hardware.
+#version 120
+
+// Input vertex attributes (from vertex shader)
+varying vec2 fragTexCoord;
+varying vec4 fragColor;
+
+// Output fragment color
+vec4 finalColor;
```

Comment:
Why is this needed?

Reply:
so it doesn't error out, when I removed it it wouldn't run.